Zend/zend_interfaces.stub.php

```diff
@@ -48,11 +48,11 @@ public function offsetUnset(mixed $offset);
 
 interface Serializable
 {
-    /** @return string */
+    /** @return ?string */
```

Comment:
```suggestion
    /** @return string|null */
```

Reply:
CS fixer prefers `?string`, `|null` for explicit doc purposes?

Reply:
and in the docs? https://github.com/php/doc-en/pull/423